src/main/webapp/scripts/map.js

```diff
@@ -49,31 +69,76 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  
   /*Add Hard-Coded Markers */
   addMarker(map);
 }
 
-/* Temporary Function to Hold Marker for Sky Garden */
 function addMarker(map) {
-  const skyGarden = {lat: 51.510881, lng: -0.083751}
-  const marker = new google.maps.Marker({
-    position: skyGarden,
+  //hardcoded marker data
+  const mildredsKingsCross = new google.maps.Marker({
```

Comment:
How do you plan on creating the markers from many many recommendations from datastore?

Reply:
I'm not sure what the best method will be, for such a large data set, but I will look into it. Right now, I haven't had the time yet, but if you have any ideas?

Reply:
not just from a large dataset, I don't think this will be possible for data fetched from the server